src/pyOpenMS/pxds/IsotopeDistribution.pxd

```diff
@@ -153,18 +169,28 @@ cdef extern from "<OpenMS/CHEMISTRY/ISOTOPEDISTRIBUTION/CoarseIsotopePatternGene
                                                                      double average_weight_fragment,
                                                                      UInt S_fragment,
                                                                      libcpp_set[ unsigned int ]& precursor_isotopes) nogil except +
+            # wrap-doc:
+                #   Estimate peptide fragment IsotopeDistribution from the precursor's average weight,
+                #   number of sulfurs in the precursor, fragment's average weight, number of sulfurs in the fragment,
+                #   and a set of isolated precursor isotopes.
 
         # Estimate RNA fragment IsotopeDistribution from the precursor's average weight,
```

Comment:
Maybe we can remove the additional comments if the same is mentioned in # wrap-doc?